frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -294,8 +298,12 @@ export const EnvFromEditor = withDragDropContext(
         <>
           <div className="row pairs-list__heading">
             {!readOnly && <div className="col-xs-1 co-empty__header" />}
-            <div className="col-xs-5 text-secondary text-uppercase">{firstTitle}</div>
-            <div className="col-xs-5 text-secondary text-uppercase">{secondTitle}</div>
+            <div className="col-xs-5 text-secondary text-uppercase">
+              {firstTitle ? firstTitle : t('environment~Config map/secret')}
```

Comment:
Why do we need defaulting now when we didn't before?

This is shorter as

```suggestion
              {firstTitle || t('environment~Config map/secret')}
```

Reply:
We had defaulting previously on 345; I wasn't sure how to i18n in that context so I moved the defaulting into the component.